### DISCUSSION TITLE
Impact of current changes in the CDS API on open-meteo API for ERA5 land?

### DISCUSSION THREAD

**Discussion by MxNl** [open-meteo/open-meteo#657] [General]
Hello,

I recently read that there are some disruptions because of CDS migration and changes of its API. Can you estimate how this affects data accesibility via open-meteo API? Thanks a lot!

**Comment by patrick-zippenfenig** [comment]
Thanks for the question. Yes, I am aware of the CDS disruptions and actively working on adapting the latest changes. Literally, yesterday at 17:30 the [CDS change announcement](https://confluence.ecmwf.int/display/CUSF/A+new+CDS+soon+to+be+launched+-+expect+some+disruptions) was updated with [migration information](https://confluence.ecmwf.int/pages/viewpage.action?pageId=392594360).... hours after the CDS APIs started having issues....

**Reply by MxNl** [reply]
Thanks for your quick reply and all your maintenance work! This is going to be a bumpy ride :-)
